api/rs/slint/Cargo.toml

```diff
@@ -37,6 +37,14 @@ default = [
 ## Enable use of the Rust standard library.
 std = ["i-slint-core/std"]
 
+## Enable the translations using [gettext](https://www.gnu.org/software/gettext/gettext)
+##
+## the `@tr(...)` code from .slint files will be transformed into call to `dgettext`
+## with the crate name as domain name
+##
+## translations must be enabled with the [`init_translations!`] macro
+gettext = ["i-slint-core/gettext-rs"]
```

Comment:
I'm curious, is there a reason for not adding this to the set of default features?

Reply:
This is a somehow heavy dependency, especially on non-linux.   Most people will not have translation as they begin a project anyway.  And we may use other features in the future to do thranslation instead of gettext.  This is my thinking.

Reply:
Okay. Thanks for the explanation.